### PR TITLE
Update build intructions for Debian/Ubuntu [skip ci]

### PR DESCRIPTION
**Pull request by crackedmind**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ What you need to have installed:
 * zlib
 * opus (libopus-dev)
 
+Debian/Ubuntu:
+```
+sudo apt-get install qtbase5-private-dev zlib1g-dev libopenal-dev libavcodec-dev libavresample-dev libswscale-dev libopenal-data libopenal1 libavutil-dev
+```
```
Comment (ForNeVeR) [inline README.md]: I would write that as
```
 ``console
 # apt-get install ...
 ``
```

Comment (crackedmind) [reply README.md]: Updated

File: README.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ What you need to have installed:
 * zlib
 * opus (libopus-dev)
 
+Debian/Ubuntu:
+```console
+sudo apt-get install qtbase5-private-dev zlib1g-dev libopenal-dev libavcodec-dev libavresample-dev libswscale-dev libopenal-data libopenal1 libavutil-dev
+```
```
Comment (ForNeVeR) [inline README.md]: Nah, it isn't a proper usage of `console`. Compare this:

```console
sudo apt-get install ...
```

and this:

```console
# apt-get install ...
```

It gets the correct coloring if starting every line with `$` or `#`.

**Review by ForNeVeR — CHANGES_REQUESTED** [review]

**Review by ForNeVeR — APPROVED** [review]
Thanks!

**Pull request merged**